css/src/components/checkbox.scss

```diff
@@ -0,0 +1,114 @@
+/* stylelint-disable selector-max-specificity, max-nesting-depth, selector-no-qualifying-type */
+
+$checkbox-border-radius: $radius-sm !default;
+$checkbox-border-width: 0.0625em !default;
+$checkbox-unchecked-border-color: $text !default;
+$checkbox-color: $primary !default;
+$checkbox-check-color: $primary-invert !default;
+$checkbox-check-hover-color: $text-subtle !default;
+$checkbox-size: 1.25em !default;
+$checkbox-icon-size: 0.75em !default;
+$checkbox-check-top: 0.1875em !default;
+$checkbox-check-width: 0.3125em !default;
```

Comment:
These three variable names should match and be either `$checkbox-icon-{rule}` or `$checkbox-check-{rule}` or `$checkbox-icon-check-{rule}`

Reply:
Oh, same for `$checkbox-checkmark-thickness` - use either `icon` or `check` or checkmark` in all of them

Reply:
Let's go with `$checkbox-check-*`, updated